Example/Tests/Core/TweakManagerTests.swift

```diff
@@ -88,11 +88,46 @@ class TweakManagerTests: XCTestCase {
         NotificationCenter.default.post(name: TweakProviderDidChangeNotification, object: self, userInfo: userInfo)
         XCTAssertFalse(didCallClosure)
     }
+    
+    func testTweakManagerDecryption() throws {
+        var mutableTweakProvider = try XCTUnwrap(tweakManager.mutableTweakProvider)
+        let feature = "password"
+        let variable = "variable"
+        
+        let encodedString = try XCTUnwrap("my secret password".data(using: .utf8)?.base64EncodedString())
+        
+        mutableTweakProvider.set(encodedString, feature: feature, variable: variable)
+        mutableTweakProvider.decryptionClosure = { tweak in
+            let data = Data(base64Encoded: tweak.stringValue!).map {
+                String(data: $0, encoding: .utf8)
+            }!
+            
+            return data!
```

Comment:
I couldn't use `XCTUnwrap` here because it's a closure, so I just resorted to force unwrap. I could change it to `?? ""` instead so it will soft fail in a worst case scenario